frontend/src/store/modules/credential.js

```diff
@@ -1,50 +1,50 @@
 const state = {
-  list: [],
+	list: [],
```

Comment:
Wonder what's the reason for the formatting change? It's getting different across the files 🤔

Reply:
yes, this is because we have not fully integrated eslint setup